### ISSUE TITLE
Contrast checker should be more colorblind-accessible, maybe with icons?

### ISSUE THREAD

**Issue by heikadog** [juxtopposed/realtimecolors#27]
**Is your feature request related to a problem? Please describe.**
The contrast checker should be more colorblind-accessible, in the same vein as Discord does. 

**Describe the solution you'd like**
There should be icons in the contrast checker, along with the red, yellow, and green that are already there.
Possible examples:
Red: X
Yellow: Dash
Green: Check

**Describe alternatives you've considered**
Obvious alternative would be not having it, but I think this would be important to have for colorblind users!

**Additional context**
This should be on by default, but it should also be able to be turned off.

Discord's example: 
![image](https://github.com/juxtopposed/realtimecolors/assets/41552394/083ca54d-0266-4579-a9bb-9a3318629710)

**Comment by sp96296** [comment]
[Here's a good related article](https://uxmovement.com/buttons/the-myths-of-color-contrast-accessibility/) on the matter (and web accessibility in general).

**Comment by Snailedlt** [comment]
Agreed. Perhaps it could be as easy as just changing the fill and stroke colors?

**Before:**
![image](https://github.com/juxtopposed/realtimecolors/assets/43886029/7793d0cd-661c-4d6b-9483-9cbbf88ed570)

**After:**
![image](https://github.com/juxtopposed/realtimecolors/assets/43886029/17a073b9-70ce-433c-a7ae-eddfcff027a8)


The icons need some work too, they are quite crude looking. especially the check mark icons. Still, just with this small color change in the svg's it's much easier to see the contrast rating at a glance

**Comment by juxtopposed** [comment]
The contrast indication with colors was available in the previous version. But some users reported it can actually conflict with their own color palette sometimes. That's mainly why color indication was removed and replaced with symbols.

**Comment by Snailedlt** [comment]
@juxtopposed I assume the issue was because there were no outlines. If it has outlines (like in the screenshot in my comment), and filled in color that shouldn't be an issue
